### PR TITLE
Fix remote switch

### LINKED ISSUE #13: Remote switch not applied
When you switch to a remote it is not applied.
You can still see the remote that was previously loaded.
You need to go to edit and then come back for the remote switch to be applied.

Huawei P30 Pro - Android 10 - v1.7.0
Also reproduceable from the tip of `libre`.

### PR DESCRIPTION
**Pull request by Slion**
Use AndroidX drawer layout.
Close #13.

### CODE REVIEW

**Review by Biswa96 — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -37,4 +37,5 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.10.1'
     implementation 'org.jsoup:jsoup:1.16.2'
     implementation project(':libraries:androidlib')
+    implementation 'androidx.drawerlayout:drawerlayout:1.2.0'
```
Comment (Biswa96) [inline app/build.gradle]: nit: could you move the dependency in alphabetic order?

**Comment by Biswa96** [comment]
Just for info. This fixes other two issues also.

1. The toolbar now shows correct remote name after opening/closing the left navigation fragment.
2. The left navigation fragment is not show when one or more remote present.

File: app/build.gradle (hunk 1 above)
Comment (Slion) [reply app/build.gradle]: Sorry, not worth my time

**Comment by Slion** [comment]
> Just for info. This fixes other two issues also.
> 
> 1. The toolbar now shows correct remote name after opening/closing the left navigation fragment.
> 2. The left navigation fragment is not show when one or more remote present.

Indeed, the issue was that the drawer open and close events did not fire and it was relying on those events to maintain the UI states. Fixing the drawer events fixed a bunch of things at once.

Thanks for merging those pull requests and patching them to your liking 🥳

**Pull request merged**